### PR TITLE
Tiktok support

### PR DESCRIPTION
**Pull request by Maxou44**
Basic support for tiktok links, it don't support short links yet (see: #66)

### CODE REVIEW

**Comment by Maxou44** [comment]
I failed, the MR also includes Dailymotion supports

File: tests/index.test.js [hunk 1]
```diff
@@ -377,3 +377,21 @@ test('microsoft stream links returns undefined id if id missing', (t) => {
...
+
+/**
+ *  TikTok should be able to find these patterns:
+ *
+ *  Urls:
+ *  https://www.tiktok.com/@brickabrackcommunity/video/6950630446614990085
+ *  https://www.tiktok.com/@brickabrackcommunity/video/6950630446614990085?lang=fr&is_copy_url=1&is_from_webapp=v1
+ *  https://www.tiktok.com/@brickabrackcommunity/video/6950630446614990085?sender_device=pc&sender_web_id=6930592755036374533&is_from_webapp=v1&is_copy_url=0
+ *
+ *  Not supported yet (requires a fetch):
```
Comment (radiovisual) [inline tests/index.test.js:389]: We should probably add a note about this to the Readme in the `Tiktok` patterns section, to make it more visible which Tiktok patterns this module doesn't support.

**Review by radiovisual — CHANGES_REQUESTED** [review]

File: tests/index.test.js (hunk 1 above)
Comment (radiovisual) [reply tests/index.test.js:396]: I added a note about the unsupported TikTok URL to the Readme via https://github.com/radiovisual/get-video-id/pull/67/commits/6d1a888239a1476d3db86251bbfd5e13c3dd1267.

**Comment by radiovisual** [comment]
@Maxou44 TikTok support was added in version `3.4.0`. Thanks for your contribution!

**Pull request merged**